src/components/AdministrationDatePicker/AdministrationDatePicker.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <div>
+    <PvPanel header="Select Administration Dates" class="mb-2" style="height: 12.5rem">
+      <div class="flex justify-content-center mb-2">
+        <PvSelectButton
+          v-model="currentMode"
+          :options="[
+            { label: 'Help me choose', value: 'presets' },
+            { label: 'I know what to select', value: 'custom' },
+          ]"
+          :allow-empty="false"
+          option-label="label"
+          option-value="value"
+          :pt="{ root: { class: 'w-full' }, pcToggleButton: { root: { class: 'w-full' } } }"
+          @change="currentModeChange"
+        />
+      </div>
+      <div v-if="currentMode === DATEPICKER_MODE.PRESETS" class="flex flex-row w-full my-2">
```

Comment:
Let's make this responsive! 

Unfortunately, as you know, Tailwind isn't fully working on this repo so we need to do a few things: 

```suggestion
      <div v-if="decision === 'presets'" class="grid grid-cols-2 gap-2 xl:grid-cols-4">
```

Then, inside the scoped script we need to override the `.grid` class: 

```css
.grid {
  display: grid !important;
  margin: 0 !important;
}
```

And finally, in `style.css` add the following: 
```scss
@media (min-width: 1280px) {
  .xl\:grid-cols-4 {
    grid-template-columns: repeat(4, minmax(0, 1fr));
  }
}
```